app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationHeader/integrationHeader.jsx

```diff
@@ -48,9 +47,11 @@ export const IntegrationHeader = (props) => {
     breadcrumbs,
   } = props;
 
+  const displayName = details.name || name;
   const isProjectIntegrationAddLimited =
     [EMAIL, SAUCE_LABS].includes(name) && availableProjectIntegrations.length > 0;
-  const { documentationLink = '' } = details;
+
+  const documentationLink = details.documentation || details.documentationLink || '';
```

Comment:
I'm not sure we should replace the `documentationLink` here with `documentation`.
As you can see below in the `integrationDescription` they are need to be used together (if I'm correctly getting your idea around `documentation` field)
![image](https://github.com/user-attachments/assets/099d9a92-1ff0-42c3-a8dd-c64a7cb0d426)
We handle plugin `description` as plain text and process `documentationLink` as a link.

Reply:
Old plugins use documentationLink for doc link, new plugins and remote plugins use a documentation field for it. We have to check above for backward compatibility.